### PR TITLE
#195: Migrate to PK2

### LINKED ISSUE #195: Migrate to PK 2
## Situation

Exasol Testcontainers still use Project Keeper 1. We need to upgrade to the latest version.

## Acceptance Criteria

1. Project Keeper Maven plug-in updated to latest 2.x

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #195.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,46 +1,19 @@
...
+    <url>https://github.com/exasol/exasol-testcontainers/</url>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <java.version>11</java.version>
-        <junit.version>5.8.2</junit.version>
-        <maven.surefire-and-failsafe.version>3.0.0-M3</maven.surefire-and-failsafe.version>
         <testcontainers.version>1.17.2</testcontainers.version>
+        <junit.version>5.8.2</junit.version>
         <gpg.skip>true</gpg.skip>
```
Comment (kaklakariada) [inline pom.xml:14]:
```suggestion
```
Already defined in parent pom

File: pom.xml [hunk 2]
```diff
@@ -1,46 +1,19 @@
...
-        <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml
-        </sonar.coverage.jacoco.xmlReportPaths>
+        <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
```
Comment (kaklakariada) [inline pom.xml:15]:
```suggestion
```
defined in parent pom

File: pom.xml [hunk 3]
```diff
@@ -166,111 +133,14 @@
...
                     </execution>
                 </executions>
-                <configuration>
-                    <modules>
-                        <module>integration_tests</module>
-                        <module>maven_central</module>
-                    </modules>
-                    <linkReplacements>
-                        <linkReplacement>
-                            LICENSE-exasol-jdbc.txt|https://docs.exasol.com/connect_exasol/drivers/jdbc.htm
```
Comment (kaklakariada) [inline pom.xml:184]: Do you need to add this to .project-keeper.yml?

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (redcatbear) [reply pom.xml:184]: The link checker says: yes.

And here I hoped that PK would do that for me. :-)

Comment (redcatbear) [reply pom.xml:184]: Fixed.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**